src/main/java/frc/robot/Robot.java

```diff
@@ -140,6 +137,8 @@ public void robotInit() {
     DRIVE_TRAIN_SUBSYSTEM.setDefaultCommand(drivetrainDefaultCommand);
     configureButtonBindings();
     configureTriggers();
+
+    GAMEPAD.getButton(ButtonCode.LEFTBUMPER).onTrue(new InstantCommand(() -> Robot.POSE_ESTIMATOR_SUBSYSTEM.resetPosition()));
```

Comment:
@Tair-Karanfili This button binding is probably okay, but we do have zeroing the gyroscope bound to LB, RB, and Y when they're all pressed. We shouldn't be using that often during matches, but if we do need to zero the gyroscope then this will also unintentionally be triggered. I don't entirely understand what this does yet, so maybe that won't be a problem (maybe check with @czbeatty). It seems like something that could also be moved to a button like `B` on the controller instead, which is unused as of right now. You can also check out the controls map in #software

You can move this line to the `configureButtonBindings()` method too

Reply:
@milligansarah changed